src/pages/benefits/benefitFormUI/BenefitApplicationForm.tsx

```
@@ -104,48 +115,102 @@ const BenefitApplicationForm: React.FC = () => {
+    // Merge application and document schemas
     const properties = {
       ...(applicationFormSchema?.properties || {}),
       ...(docSchemaData?.properties || {}),
     };
+    console.log("properties", properties);
+
+    // Collect required fields
     const required = Object.keys(properties).filter((key) => {
       const isRequired = properties[key].required;
       if (isRequired !== undefined) {
         delete properties[key].required;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)